### ISSUE TITLE
Admonishments inside lists restarts numbering

### ISSUE THREAD

**Issue by mattburgess** [tommilligan/mdbook-admonish#123]
I have the following Markdown:

````markdown
1. Thing one

   ```sh
   $ echo "Thing one"
   ```

1. Thing two

   ```sh
   $ echo "Thing two"
   ```

1. Thing three

   ```sh
   $echo $Thing three"
   ```

   ```admonish warning
   **Do not** run this
   ```

1. Thing 4

   ```sh
   $ echo "Thing four"
   ```
````

This doesn't quite get rendered correctly:

1. The admonishment isn't indented to align with the list item's text; instead it's aligned with the list numbers.
2. "Thing 4" gets rendered in its own `<ol>` resulting in the numbering starting from 1 again, instead of it continuing at 4.

**Comment by tommilligan** [comment]
Hi, thanks for reporting this issue! I believe I've found the underlying cause - a fix is just running through CI now: #124

If you're able to, please install that branch directly and see if it fixes your use case:

```
cargo install --git https://github.com/tommilligan/mdbook-admonish --branch bug-123
```

**Comment by mattburgess** [comment]
Wow, what a fantastically quick response. Works perfectly here. Thanks very much!
